src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -413,7 +413,10 @@ export default function createPostGraphileHttpRequestHandler(
         )
       : null;
 
```

Comment:
We need to add some validation of this option somewhere. This is almost certainly not the best place...

```suggestion

    if (options.websocketOperations && ['all', 'subscriptions', 'none'].indexOf(options.websocketOperations) === -1) {
      throw new Error("Invalid websocketOperations argument");
    }
```